gtwrap/template_instantiator.py

```diff
@@ -272,28 +273,27 @@ def __init__(self, original, instantiations=(), new_name=''):
         self.parent_class = original.parent_class
         self.parent = original.parent
 
-        if not original.template:
-            self.name = original.name
-            self.ctors = list(original.ctors)
-            self.static_methods = list(original.static_methods)
-            class_instantiated_methods = list(original.methods)
-            self.properties = list(original.properties)
-        else:
+        if original.template:
```

Comment:
Now that you know what this code does, and logic below, add a comment here to that effect.